docs/source/concepts/HowToDefineGeometricShape.rst

```diff
@@ -340,6 +452,59 @@ axis runs from the start aperture to the end aperture. "Height" is along
 the y-axis and "width" runs along the x-axis, before the application of
 the "axis" rotation.
 
+Rotating Shapes
+~~~~~~~~~~~~~~~
+
+*Note that some shapes (such as* `Cylinder (finite height) <Cylinder_>`_ *and* `Tapered Guide`_ *) can be oriented in a
+certain direction, using the* ``axis`` *tag, but this does not support all possible 3D rotations.*
```

Comment:
I suggest you extend to say "... using the ``axis`` tag, but this representation isn't general enough to support all possible 3D rotations for all shapes". 
I think the axis tag does actually allow all possible 3D rotations for the cylinder. It is just for shapes that aren't symmetrical about a single axis where a single axis tag wasn't going to work